frontend/src/components/common/IconButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Box, Button, IconButton as ChakraIconButton } from "@chakra-ui/react";
+
+interface IconButtonProps {
+  color: string;
+  hoverColor: string;
+  onClick: () => void;
+  icon: React.ReactElement;
+}
+
+const IconButton = ({
+  onClick,
+  color,
+  hoverColor,
+  icon,
+}: IconButtonProps): React.ReactElement => {
+  return (
+    <Box _hover={{ color: hoverColor }} color={color}>
```

Comment:
should we set default `color="grey.300"` and `hoverColor="blue.100"` so we don't have to pass it in every time?

Reply:
i think there's two more in `QuestionCard` you can refactor and also `DisplayClassroomsPage`